google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineDeploymentConfigurationPanel.java

```diff
@@ -123,6 +128,12 @@ public void applyEditorTo(@NotNull AppEngineDeploymentConfiguration configuratio
     configuration.setPromote(promoteCheckbox.isSelected());
     configuration.setCloudProjectName(projectSelector.getText());
     configuration.setStopPreviousVersion(stopPreviousVersionCheckbox.isSelected());
+    configuration.setDeployAllConfigs(deployAllConfigsCheckbox.isSelected());
```

Comment:
I need to test what actually happens in the UI but I believe deploy all configs should apply only to standard (and be hidden from flex) which is why this was not previously in the common editor class.